ci-operator/step-registry/azure/provision/service-principal/minimal-permission/azure-provision-service-principal-minimal-permission-commands.sh

```diff
@@ -200,6 +200,7 @@ ${required_permissions}
 if [[ "${CLUSTER_TYPE_MIN_PERMISSOIN}" == "IPI" ]]; then
     required_permissions="""
 \"Microsoft.Compute/availabilitySets/write\",
+\"Microsoft.Compute/availabilitySets/delete\",
```

Comment:
we don't need to add this permission on UPI?

Reply:
Actually it's needed, I see write only added on IPI, so just add delete on IPI. On UPI ci, we can't create machineset automatically, so I think it's ok to only add it on IPI.
For the upi clusters, if we want to create/delete machineset manually, these permissions are needed.